### PR TITLE
fix (client): solve migration bug with indexes

### PR DESCRIPTION
**Pull request by kevin-dp**
This PR solves the index bug described in #457.
The bug was caused by the client migration code which tried to build table triggers even if the table information is missing. The code was modified not to build any triggers if the table information is missing. The table information may be missing e.g. when only creating an index.

### CODE REVIEW

File: .changeset/funny-terms-knock.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"electric-sql": patch
+---
+
+Do not build triggers if table information is missing.
```
Comment (alco) [inline .changeset/funny-terms-knock.md:5]: I may be nitpicking here but I think these changelog entries should make sense to anyone looking at the release notes when they are upgrading to a new version.

As is, this description does not contain any more info than the commit message itself.

I would maybe rephrase it as

> Fix the bug where the client would crash/stop working/stop syncing if it received a migration containing a new index creation.

File: clients/typescript/src/migrators/builder.ts [hunk 2]
```diff
@@ -74,6 +74,9 @@ export function makeMigration(migration: MetaData): Migration {
     .map((op) => op.stmts.map((stmt) => stmt.sql))
     .flat()
   const tables = migration.ops
+    // if the operation did not change any table
+    // then ignore it as we don't have to build triggers for it
+    .filter((op) => op.table !== undefined)
```
Comment (balegas) [inline clients/typescript/src/migrators/builder.ts:79]: @kevin-dp it's approved. up to you to add a test before merging
